app/routes/account/billing/invoices/list.js

```diff
@@ -12,6 +12,12 @@ export default class extends Route.extend(EmberTableRouteMixin) {
         return this.l10n.t('Due');
       case 'upcoming':
         return this.l10n.t('Upcoming');
+      case 'unpaid':
+        return this.l10n.t('Unpaid');
+      case 'refunding':
+        return this.l10n.t('Refunding');
+      case 'refunded':
+        return this.l10n.t('Refunded');
```

Comment:
Simply use capitalize function instead of switch case. Check out `admin/sessions/list` route. Use `All` in else

Reply:
ok